inference_lib/README.md

```diff
@@ -0,0 +1,252 @@
+# SpQR model compression
+
+
+**Note:** This repository contains quantization algorithm and the model evaluation code for SpQR method for LLM compression; 
+The efficient inference code will be added soon.
```

Comment:
Isn't this readme for inference code ?:)

Reply:
Resolved by reworking the README